mobilitypath_publisher/src/mobilitypath_publisher.cpp

```diff
@@ -106,39 +105,45 @@ namespace mobilitypath_publisher
 
     cav_msgs::Trajectory MobilityPathPublication::TrajectoryPlantoTrajectory(const std::vector<cav_msgs::TrajectoryPlanPoint>& traj_points, const geometry_msgs::TransformStamped& tf) const{
         cav_msgs::Trajectory traj;
-        cav_msgs::LocationECEF ecef_location = TrajectoryPointtoECEF(traj_points[0], tf);
+        cav_msgs::LocationECEF ecef_location = TrajectoryPointtoECEF(traj_points[0], tf); //m to cm to fit the msg standard
```

Comment:
According to the message spec this first point is in meters and only the offsets are in cm.